### PR TITLE
completed gear management

### PR DESCRIPTION
**Pull request by VaidaKa**
Completed basic gear management with very basic frontend. 

Screenshot 2024-05-31 at 23 04 56
home page with links to create new listing and with link to view all gear.  Navbar HOME button fixed with correct path. 

Screenshot 2024-05-31 at 23 05 47
all gear page /gears

Screenshot 2024-05-31 at 23 08 04
creating new gear 

view, edit, update functions working. 
I need help with delete action (not deleting the gear)

To bear in mind that we don't have database seeded properly - all gear will show on page only when you create one yourself

### CODE REVIEW

File: app/controllers/gears_controller.rb
Comment (VaidaKa) [inline app/controllers/gears_controller.rb:1]: completed gear CRUD actions

File: app/views/gears/_form.html.erb
Comment (VaidaKa) [inline app/views/gears/_form.html.erb:1]: Created _form to render edit and new html

File: app/views/gears/index.html.erb
Comment (VaidaKa) [inline app/views/gears/index.html.erb:1]: updated gears index with basic frontend

File: app/views/gears/new.html.erb
Comment (VaidaKa) [inline app/views/gears/new.html.erb:1]: render from _form

File: app/views/gears/show.html.erb
Comment (VaidaKa) [inline app/views/gears/show.html.erb:1]: updated with basic frontend

File: app/views/pages/home.html.erb
Comment (VaidaKa) [inline app/views/pages/home.html.erb:1]: added a basic container to home page for Jurij to start frontend

File: app/views/shared/_navbar.html.erb
Comment (VaidaKa) [inline app/views/shared/_navbar.html.erb:1]: to be able to use Home button in nabber updated root path so it would work to return to home page

**Review by yaroslavgubich — APPROVED** [review]

**Pull request merged**